models/article_model.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db/connection");
+
+exports.selectArticles = (id) => {
+  
+           return db.query (  `SELECT articles.*, COUNT(comments.article_id) AS comment_count
+           FROM articles 
+          JOIN COMMENTS
+           ON articles.article_id = comments.article_id 
+           WHERE articles.article_id = $1
+           GROUP BY articles.article_id
+           ;`, [id])
```

Comment:
So this query can just be `SELECT * from articles where article_id = $1`

There is no need for a JOIN of any kind. You have all the data needed inside the article table - dont need the comment count for this ticket.